tests/usecase/test_sap_hana.py

```diff
@@ -1,5 +1,7 @@
 import json
 import os
+import urllib.parse
```

Comment:
Why are we changing SAP/Hana code in a PR that doesn't relate to SAP/Hana?

Reply:
These tests were flaky, relying on dictionary ordering (which they shouldn't), revealed by pinning the Python version to 3.4.3

Reply:
Okay, cool.

Let's make sure that someone who is familiar with the code (@cwesdorp going from git blame) has reviewed this before we merge, since these test changes seem functional (i.e. supplied URLs are checked differently).